### DISCUSSION TITLE
v-t decrepetation warning after 11.0.0 in console but no usages can be found

### DISCUSSION THREAD

**Discussion by pzgz** [intlify/vue-i18n#2055] [Q&A]
After upgraded to v11.0.0. Got following console message, pretty annoying, but the strange part is, there's no `v-t` unsage in my project. Any hints I can locate where's the cause?

> [intlify] 'v-t' has been deprecated in v11. Use translate APIs ('t' or '$t') instead.

![CleanShot 2024-12-25 at 14 23 31@2x](https://github.com/user-attachments/assets/4f1f91fd-5a47-471c-96b1-4785d756a559)

**Comment by Safirion** [comment]
I don't use v-t in my project either but I get this warning too. 🤔

Moreover
![image](https://github.com/user-attachments/assets/197bd0b3-47fa-4cf8-bc5c-a3c1fe20ff2c)

I may have misunderstood the vue-i18n documentation but does this mean that the vuejs option API will no longer be supported by this plugin in next releases ?

Also, I don't understand why there is mention of "legacy" vs "composition" mode in vue-i18n. Since the option API  is not deprecated in vuejs, it seems essential to me to keep it working in vue-i18n.

**Reply by pzgz** [reply]
by checking on source codes, it looks like when the warning message was triggered when `v-t` has been registered as a directive. It's not triggered when the directive been used actually. That's why I am proposing a way to opt-out `v-t` directive by user, or move the warning message from register into usage. But not sure if it's worthing efforts for a decrepeting feature

**Comment by pzgz** [comment]
repo owner has fixed the issue in 11.0.1, thanks a lot
